### ISSUE TITLE
[Bug]: Error during building - jandex-gradle-plugin 

### ISSUE THREAD

**Issue by DudekJakub** [kordamp/jandex-gradle-plugin#30]
**Java version**
java version 21.0.1 Amazon Corretto

**Gradle version**
8.5

**Operating system**
MacOS

--------

Hello guys.

I would like to raport that I have the same issue as this one here [bug-jandex-maven-plugin](https://github.com/apache/hop/issues/3637) while use **jandex-gradle-plugin**. 

From what I see, maven plugin has been fixed and now uses 
`<jandex.version>3.1.6</jandex.version>`
while gradle plugin still uses old version of jar 
`jandexVersion = 3.0.5`

As the issue is exactly the same, my question is: are you planning to upgrade jar version with proper fix? Because currently while using gradle-plugin we are forced to set version to 3.1.6 with this block:
```
jandex {
    version.set("3.1.6")
}
```

Thank you for any response!

**Comment by cthiebault** [comment]
For people using the Gradle Kotlin DSL:
```kotlin
project.configure<org.kordamp.gradle.plugin.jandex.JandexExtension> {
  version = "3.1.6"
}
```

**Comment by aalmiray** [comment]
Fixed with https://github.com/kordamp/jandex-gradle-plugin/commit/afe1c8247805e0aaf3aa9cf6528ea1df877d4f08

**Comment by aalmiray** [comment]
🎉 This issue has been resolved in v2.0.0 ([Release Notes](https://github.com/kordamp/jandex-gradle-plugin/releases/tag/v2.0.0))
